docs/package.json

```diff
@@ -16,9 +16,9 @@
     "pcln-modal": "^2.0.4",
     "pcln-slider": "^2.2.0",
     "prop-types": "^15.6.2",
-    "react": "^16.4.1",
-    "react-dom": "^16.4.1",
-    "styled-components": "^3.4.10"
+    "react": "^16.10.2",
+    "react-dom": "^16.10.2",
```

Comment:
Awesome! I created https://bitbucket.corp.priceline.com/projects/CC/repos/pcln-react-hooks/browse the other day, still need to work on finishing the build process / publishing but I'm hoping to see more hook usage starting.

Reply:
awesome. this only sets the dev dep, for now. hooks as a peer will probably have to wait for the next semver major

Reply:
For sure and I can't imagine to much use in the DS but this will encourage consuming apps to upgrade and use hooks 😄